acceptance/tests/verify_mountpoints.rb

```diff
@@ -0,0 +1,41 @@
+# This test is intended to demonstrate that mount resource can mount tmpfs directory
+# and the mount facter mountpoints shouls show the mount on tmpfs
+
+test_name "FACT-1052 - C98163 mountpoints fact should show mounts on tmpfs" do
```

Comment:
Ticket number should be 1502.

And out of curiosity, what does the C-number mean?